pom.xml

```diff
@@ -82,9 +82,6 @@
           <value>!true</value>
         </property>
       </activation>
-      <modules>
-        <module>qa</module>
-      </modules>
     </profile>
```

Comment:
❌ ❓ Why was this removed? This is necessary to execute the QA module.

Reply:
This `qa` module was not used at the end, for some reason only TestStandaloneCamunda was there, but was not being reading correctly by the build. I was looking the classes and the clustering tests was placed in the zeebe folder (following the second comment).

Did I miss something on this changes @Zelldon ?